pkg/cli/mirror/copy.go

```diff
@@ -783,7 +749,7 @@ func pushImage(from, to string, dstSkipTLS bool, insecurePolicy bool, funcs Remo
 	}
 
 	// call the copy.Image function with the set options
-	_, err = funcs.push(ctx, policyContext, destRef, srcRef, &imagecopy.Options{
+	_, err = funcs.copy(ctx, policyContext, destRef, srcRef, &imagecopy.Options{
```

Comment:
Can you change the ForceManifestMIMEType to be v2s2 like this:

```
		ForceManifestMIMEType: manifest.DockerV2Schema2MediaType,
```

Basically we'd prefer to use OCI as the image layout, but use v2s2 media types when images are saved in the file system. As it is configured, the copy action is saving as OCI media types and OCI image layout. 

We plan to generate catalog images on the filesystem in OCI layout with v2s2 media types, and let oc mirror process these in the "mirror action".

Reply:
@deejross 
Are there any impacts forcing the mime type back to v2s2?

Reply:
Not that I'm aware of, so long as the writer can output the desired format.

Reply:
Hi @deejross @jchunkins 
I'm not sure if this fits in the long run for all the users of oc-mirror who have oci catalogs.
For now, our strategy was not to interfere with the content of the images, so that the digest doesn't change. 
Wouldnt `ForceManifestMIMEType: manifest.DockerV2Schema2MediaType,` rewrite the manifest content and make the digest change?
Do you think we should introduce this as a separate option?

Reply:
Thanks @jchunkins 
We can merge this PR like this and eventually create a new story for a specific flag to specify the manifest media type that needs to be used (if we can achieve this with the libraries used)